Off-Top-App/SessionHandlerService
# OT-SessionHandler | Migrated the websocket and the required Consumers and Producers

**WHAT I DID:**
1.  Migrated the websocket and the required Consumers and Producers that are required to handle focus scores and audio data.

HOW TO TEST:
1. Have Kafka/zookeeper, [UserService](https://github.com/Off-Top-App/UserService) & off-top-flutter running
2. Run the app by using `mvn spring-boot:run` and be able to receive data from the WebSocket inside of SessionHandler by recording audio from flutter. More info can be found from the [documentation](https://docs.google.com/document/d/1nK7zDGFb-LmKLKMDAnWmciQzU2fMDnsbb6lOZ60YYU4/edit#) here; it is very similar to off-top-java.
3. Make sure you can read data being produced in Producer.py which is located in [off-top-python](https://github.com/Off-Top-App/off-top-python)/backend-services/Services/Kafka/Producer.py
You will need to change the user_id on line 23 to user_id value associated within your MySQL DB of offTopUser, which is now connected to UserService instead of off-top-java.
